docs/reference/tags/package_step_product_search_tag/index.md

```diff
@@ -64,9 +64,28 @@ The tag returns an `items` array and a `Paginate` pagination object to the block
 
 ## Search Params
 
-#### include_organisation_products
```

Comment:
This is still used in the tag itself, right: https://github.com/easolhq/easol/blob/main/app/tags/package_step_product_search_tag.rb#L32? it's just its effect is different now 🤔